x-pack/solutions/security/plugins/security_solution/public/flyout/shared/hooks/use_on_expandable_flyout_close.test.tsx

```diff
@@ -21,7 +21,7 @@ describe('useOnExpandableFlyoutClose', () => {
 
     window.removeEventListener = jest.fn().mockImplementationOnce((event, callback) => {});
 
-    renderHook(() => useOnExpandableFlyoutClose({ callback: callbackFct }));
+    const { unmount } = renderHook(() => useOnExpandableFlyoutClose({ callback: callbackFct }));
```

Comment:
fix [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/267344#019469ff-7fb9-4c5d-8569-2e445aab27be) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/267344/jobs/019469ff-7fb9-4c5d-8569-2e445aab27be/artifacts/01946a1c-62fa-4ee7-a99f-140fb2dd6541) Jest Tests #9 / useOnExpandableFlyoutClose should run the callback function and remove the event listener from the window

I just assume this test wasn't testing what was intended. It might be that removeEventListener was called on the initial render during effect re-rerun and that is what the assertion was cathing.